sunpy/map/tests/test_mapbase.py

```diff
@@ -66,6 +66,19 @@ def aia171_test_map_with_mask(aia171_test_map):
     return sunpy.map.Map(np.ma.array(aia171_test_map.data, mask=mask), aia171_test_map.meta)
 
 
+@pytest.fixture
+def simple_map():
+    # A 2x2 map, with it's center at (0, 0), and scaled differently in
+    # each direction
+    data = np.arange(4).reshape((2, 2))
+    ref_coord = SkyCoord(0, 0, frame='helioprojective', obstime='now', unit='deg')
+    ref_pix = [1.5, 1.5] * u.pix
```

Comment:
Isn't this inconsistent with #4154? If the reference pixel is now zero-based, shouldn't this be (0.5,0.5)? Or is the input to `make_fitswcs_header` expected to be one-based?

Reply:
yeah this now needs updating post-#4154

Reply:
After drawing pictures for 10 minutes, really had a crisis and thought I understood nothing until I realized we switched conventions 😆